test/Datadog.Trace.ClrProfiler.IntegrationTests/AWS/AmazonSQSTests.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections.Generic;
+using System.Linq;
+using Datadog.Trace.TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.AWS
+{
+    public class AmazonSqsTests : TestHelper
+    {
+        private readonly List<AmazonSqsExpectation> _expectations = new List<AmazonSqsExpectation>();
+
+        public AmazonSqsTests(ITestOutputHelper output)
+            : base("Amazon.SQS", output)
+        {
+            _expectations.Add(CreateExpectation("CreateQueue"));
+            _expectations.Add(CreateExpectation("ListQueues"));
+            _expectations.Add(CreateExpectation("GetQueueUrl"));
+            _expectations.Add(CreateExpectation("SendMessage"));
+            _expectations.Add(CreateExpectation("SendMessage"));
+            _expectations.Add(CreateExpectation("ReceiveMessage"));
+            _expectations.Add(CreateExpectation("DeleteMessage"));
+            _expectations.Add(CreateExpectation("SendMessageBatch"));
+            _expectations.Add(CreateExpectation("SendMessageBatch"));
+            _expectations.Add(CreateExpectation("ReceiveMessage"));
+            _expectations.Add(CreateExpectation("DeleteMessageBatch"));
+            _expectations.Add(CreateExpectation("PurgeQueue"));
+            _expectations.Add(CreateExpectation("DeleteQueue"));
+        }
+
+#if NETCOREAPP2_1
```

Comment:
Is there a reason this is conditional on netcoreapp2.1? I would imagine some of the expectations above might be, since the .NET Core API doesn't include synchronous operations.

Reply:
The test now runs for framework